src/watchtowers/initialization.md

```diff
@@ -0,0 +1,58 @@
+(watchtower-initialization)=
+# Watchtower Initialization
+
+```{needget}
+
+* An SD card with a minimum size of 32 GB.
+
+* A computer running **Ubuntu OS** (for flashing the SD card), an internet connection, an SD card reader, and at least 16 GB of free space.
+
+* Duckietown Shell, Docker, and other configurations as specified in [Duckiebot Laptop Setup](book-opmanual-duckiebot:laptop-setup).
+
+* A Duckietown Token set up as described in [Duckiebot Account Configuration](book-opmanual-duckiebot:dt-account).
+
+---
+* A ready Watchtower.
+```
+
+## Flashing the SD card
+
+The image setup procedure for Watchtowers is the same as for Duckiebots.
+
+In the Autolab of ETH Zurich, we use the following naming convention:
+
+- Linux username: `mom`
+- Hostname: `watchtowerXX` (where `XX` specifies the number of the Watchtower)
+- Password: `MomWatches`.
+
+````{attention}
+
+Please add `--type watchtower` to the flashing procedure.
+
+```{todo}
```

Comment:
@afdaniele have we moved to `ente` for this?

Reply:
You can get rid of this line.